tests/integration/nodes/tools/test_python.py

```diff
@@ -98,7 +97,7 @@ def run(input_data):
         'pi_used': math.pi
     }
 """
-    python_node = Python(code=python_code)
+    python_node = Python(code=python_code, use_multiple_params=True)
```

Comment:
Please add tests for both cases use_multiple_params=True/False, not just update existing